tests/acceptance/expected-failures-graphAPI-on-OCIS-storage.md

```diff
@@ -0,0 +1,1671 @@
+## Scenarios from ownCloud10 core API tests that are expected to fail with OCIS storage while running with the Graph API
+
+The expected failures in this file are from features in the owncloud/core repo.
+
+TODO: group list of expected failures to their respective issues.
```

Comment:
I guess that the failures should be similar to `expected-failures-API-on-OCIS-storage.md`

Maybe you can just copy that and then make a few changes for any extra/different failures.